qa/tasks/vstart_runner.py

```diff
@@ -773,6 +766,26 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
+        # NOTE: When a command <args> is launched with sudo, two processes are
+        # launched, one with sudo in <args> and other without. Make sure we
+        # get the PID of latter one.
+        i, _break = 0, False
+        while i < 5:
+            time.sleep(5)
+            try:
+                sock = self.find_admin_socket()
+            except RuntimeError:
+                pass
+            pid = sock.replace('.asok', '')
+            pid = pid[pid.find(self.client_id + '.') + 2 : ]
```

Comment:
Will the client_id possible be a number >= 10 ?
How about `pid = re.search(r'client\.\d+\.(\d+)\.asok$', sock).group(1)`

Reply:
It can be but mostly it's just 0 and 1, since most tests use only 2 clients.